client/src/hooks/useTryCatch.tsx

```diff
@@ -11,7 +11,7 @@ interface TryCatchState {
 }
 export const useTryCatch = (): TryCatchState => {
   const [pending, setPending] = useState(false)
-  const [initializing, setInitializing] = useState(true)
+  const [initializing, setInitializing] = useState(false)
```

Comment:
need to keep as true otherwise some elements are interactive from the start

Reply:
Okay

Reply:
Done